libr/egg/egg.c

```diff
@@ -48,7 +48,7 @@ R_API REgg *r_egg_new() {
 	if (!egg->syscall) {
 		goto beach;
 	}
-	egg->rasm = r_asm_new ();
+	egg->rasm = r_asm_new (NULL);
```

Comment:
uhm thats an issue because it will be inconsistent. you should create that instance from outside or pass it as argument like in the other constructor. but all the class constructors in r2 get 0 parameters because they should not have side effects

Reply:
ok 908b0e042e935b2324aa699c6229adf83d47b5bd